README.md

```diff
@@ -10,6 +10,38 @@ Debezium is a distributed platform that turns your existing databases into event
 
 Debezium is open source under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+# Purpose of the Debezium UI
+The Debezium UI provides standalone web application, which connects to Kafka Connect via its REST API. See the [Debezium UI docs](https://debezium.io/documentation/reference/operations/debezium-ui.html) for more information
+
+# Debezium environment vars
+The table shows the ENV vars for the [Debezium UI container image](https://hub.docker.com/r/debezium/debezium-ui) and it's related `application.properties` parameter name when running the java application locally.
+
+<table id="debezium">
+  <thead>
+    <tr>
+      <th>ENV vars</th>
+      <th>application.properties vars</th>
+      <th>Default value</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td valign="top">DEPLOYMENT_MODE</td>
+      <td valign="top">deployment.mode</td>
+      <td valign="top">default</td>
+      <td valign="top">Specifies how connector properties validation mode is enabled/disabled when configuring the connectors in the Debezium UI <br><br><code>default</code>: When set to default it turns the validation mode on for the frontend<br><br> <code>validation.disabled</code>:  When set to validation.disabled it turns the validation mode off for the frontend<br>
```

Comment:
```suggestion
      <td valign="top">Specifies how the Debezium UI is deployed.<br>For example, in some environments it might not be possible to reach the underlying backend, Kafka Connect REST interface or databases, then the deployment mode can be switched to match the underlying infrastructure.
      <br><br><code>default</code>: The default deployment mode. It uses the Debezium UI backend with the configured Kafka Connect clusters via the Kafka Connect REST interface (see <a href="#KAFKA_CONNECT_URI">KAFKA_CONNECT_URI</a> how they are configured).
      <br><br> <code>validation.disabled</code>: When set to validation.disabled the UI frontend will not call the backend to validate the user input nor check the availability and proper configuration of database connections. That mode is used to only generate the Debezium connector JSON configuration without the UI backend validation.<br>
```